### PR TITLE
Improve XML documentation of AddEventLog() without LogLevel

### PR DESCRIPTION
**Pull request by Mertsch**

### CODE REVIEW

File: src/Microsoft.Extensions.Logging.EventLog/EventLoggerFactoryExtensions.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ public static ILoggerFactory AddEventLog(this ILoggerFactory factory, LogLevel m
         }
 
         /// <summary>
-        /// Adds an event logger that is enabled for <see cref="LogLevel"/>s of minLevel or higher.
+        /// Adds an event logger. Use <paramref name="settings"/> to enabled for specific <see cref="LogLevel"/>s.
```
Comment (JunTaoLuo) [inline src/Microsoft.Extensions.Logging.EventLog/EventLoggerFactoryExtensions.cs]: Can you rephrase to `Use <paramref name="settings"/> to enable logging for specific <see cref="LogLevel"/>s.`

**Comment by JunTaoLuo** [comment]
@Mertsch can you make an update and then I'll merge.

**Comment by JunTaoLuo** [comment]
Merged to dev after minor grammar update.

**Comment by Mertsch** [comment]
@JunTaoLuo Thanks for the update, sounds better that way :)
